extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -164,6 +174,23 @@ class AmpLightbox extends AMP.BaseElement {
   }
 
   /**
+   * reschedule layout on animation/transition end. But ignore the first transitionend event
+   * @param {!Event} event
+   */
+  reschedule_(event) {
+    if (event.type == 'transitionend') {
+      // Ignore the first transitionend event.
```

Comment:
`amp-lightbox` by default apply `transition: 'opacity 0.1s ease-in'` to its element. I am trying to ignore this specific `transitionend` event here.